app/views/admin/tutorials/edit.html.erb

```diff
@@ -4,11 +4,10 @@
   <p>Drag and drop to sort. Don't forget to save.</p>
   <div data-controller="tutorials">
     <%= link_to "Save Video Order", "#", "data-action" => "click->tutorials#updateOrder", class: "video-position-submit btn btn-outline", id: @tutorial.id %>
-    <%= link_to "Add Video", "#", "data-action" => "click->tutorials#showVideoForm", class: "video-position-submit btn btn-outline", id: @tutorial.id %>
   </div>
 
-  <div class="mt1" id="new-video-form">
-    <h3>New Video</h3>
+  <div class="mt1">
+    <h3>Add Video</h3>
```

Comment:
Remove New video form id and change view to only have Add Video to remove unnecessary button